omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -391,53 +383,6 @@
           <CreateFolder/>
         </Component>
       </Directory>
-
-      <Directory Id="TUFMETALOCATION" Name="repositories">
```

Comment:
We need to check with @derekwbrown if we can just remove this with no side-effects across upgrades, etc. I _think_ it's fine, but let's confirm with him.